mgmtfn/mesosplugin/README.md

```diff
@@ -0,0 +1,4 @@
+# Contiv Networking for Apache MESOS (http://mesos.apache.org/) 
```

Comment:
Can we remove this file if there isn't anything in it?
Or provide a pointer to the getting started guide in vagrant/mesos-cni directory..